tests/CMakeLists.txt

```diff
@@ -100,32 +31,26 @@ include_directories("${CMAKE_BINARY_DIR}/include"
 ###################################################################################################
 # - library paths ---------------------------------------------------------------------------------
 
-link_directories("${CMAKE_CUDA_IMPLICIT_LINK_DIRECTORIES}" # CMAKE_CUDA_IMPLICIT_LINK_DIRECTORIES is an undocumented/unsupported variable containing the link directories for nvcc
+link_directories("${CMAKE_CUDA_IMPLICIT_LINK_DIRECTORIES}" 
```

Comment:
If you use `target_link_library` with the `CUDA::cudart` or `CUDA::cudart_static` it will automatically handle this, so you should remove this.